server/models.py

```diff
@@ -247,6 +247,11 @@ def __init__(
     def __repr__(self):
         return f"<id {self.id}>"
 
+    def get_follower_josn(self) -> Dict[str, Any]:#

```

Comment:
this is a minor thing, but can you fix the spelling lol? to get_follower_json?

Reply:
fixed in [954f8ad](https://github.com/giancarlopernudisegura/cmput404/pull/100/commits/954f8ad25f870672d3c1888ec83b5ee261275443)